ampersand-collection.js

```diff
@@ -167,6 +168,8 @@ assign(Collection.prototype, AmpersandEvents, {
     get: function (query, indexName) {
         if (query == null) return;
         var index = this._indexes[indexName || this.mainIndex];
+        if(index && index[query[this.mainIndex]] && query[this.mainIndex] === undefined)
+        return;
```

Comment:
For style purposes, can you place `return` at the end of the previous line?

Reply:
Added return at the end of previous line.